test/runtests.jl

```diff
@@ -35,6 +35,16 @@ end
   end
 end
 
+@testset "Test unbounded below" begin
+  @testset "$fun" for fun in (R2, fomo, lbfgs, tron, trunk)
+    f(x) = -exp(x[1])
+    nlp = ADNLPModel(f, [2.])
+
+    stats = eval(fun)(nlp)
+    @test stats.status == :unbounded
```

Comment:
Would it make sense to also test the value of the objective function store in `stats`? Something like
```
@test stats.objective <= -1e15
```
or something like that.

Reply:
I think it would, I'll add that to the tests.